src/components/profileImage/ProfileImage.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+
+interface Props {
+  src?: string;
+}
+
+const ProfileImage = ({ src }: Props) => {
+  return (
+    <EmotionWrapper>
+      <Image
+        src={src ?? "/images/profile-image-default.png"}
```

Comment:
요기 랜덤한 기본 이미지 보여주기로 했는데, 그거 맞춰서 살짝 수정해두면 연결할때 편할 것 같습니다

```suggestion
        src={src ?? `/images/profile-image-default-${defaultPhotoId}.png`}
```

Reply:
API 문서에 1차로 유저 부분은 작성되어 있긴 해서, 그거 참고하셔도 될 듯 합니다

Reply:
@hoon5083  아 맞네요 죄송합니다. 그 랜덤으로 6개 (?) 정도 암튼 돌아가면서 랜덤하게 생성되기로 한거 그거에 대응해야겠네요. 알려주셔서 감사합니다 🙏

Reply:
@hoon5083  대응해 보았습니다!

Reply:
차라리 그냥 아무 props 도 받지 않게 하는 것도 방법일 거 같습니다. 

인증과 유저데이터 한번더 프로필 내부에서 받아오고, 생각해보니 이 컴포넌트가 props 를 받을 이유가 없는 것 같은데요? ㅎ

Reply:
그러게요 그게 더 나을 것 같습니다! 어차피 SWR 쓰니까 알아서 캐싱하고 요청 날려주기도 하고...

Reply:
말씀하신대로 변경하신거 확인하면 바로 resolve하겠습니다 :)

Reply:
@hoon5083  오 확인이 늦어 죄송합니다! 수정 하여 다시 push 하였습니다. 

만드는 과정 중에 데이터 같은 경우 내부적으로 받으면 되지만, 프로필 사진을 다양한 크기로 이용하고 싶은 경우를 고려하여 `size` prop (옵셔널, 기본값 30, (가로/세로 값으로 이용)) 을 추가했습니다. (마이페이지에는 큰 크기로 사용하여야 하는 점 고려)

Reply:
확인했습니다!